### ISSUE TITLE
508-defect-0 ❗️ Launchblocker: Smooth scroll should not be used with accordions

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#22286]
# [508-defect-0 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-0)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a user with vestibular disorder, I don't want the page to unexpectedly scroll without my permission.

## Details

Smooth scrolling that hijacks control from the user may be harmful to those with vestibular disorders.

## Acceptance Criteria

- [ ] Accordions do not smooth scroll on the page

## Steps to Recreate

Homepage accordions

## Proposed Solution (if known)

- Remove smooth scroll from the accordions OR 
- do not scroll at all (I think the design system accordion does not scrolljack the user) OR
- use the reduce motion CSS media query to make it conditional based on the user's settings

## Screenshots or Trace Logs

https://user-images.githubusercontent.com/14154792/112846795-6cbf1400-9074-11eb-989e-e214112d25f0.mov

**Comment by artsymartha68** [comment]
Related? https://github.com/department-of-veterans-affairs/va.gov-team/issues/6844

**Comment by artsymartha68** [comment]
Also related? https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/312

**Comment by artsymartha68** [comment]
@bkjohnson offered to grab this one. 

Slack discussion for reference, in case it helps. 
https://dsva.slack.com/archives/C8E985R32/p1591299017158700?thread_ts=1591298527.158400&cid=C8E985R32

@Mottie said:
Tell your devs to add this setting when the component initializes (ComponentDidMount if you're using Classes)

window.VetsGov.scroll = {
  duration: 0,
  delay: 0,
  smooth: false
};

**Comment by joshkimux** [comment]
Just tested the accordions and this looks like it's been fixed!
Thank you @bkjohnson 🥳 
https://user-images.githubusercontent.com/14154792/112902192-c85cc200-90b3-11eb-8843-8bf163f68ccc.mov
